cmd/dep/vndr_importer_test.go

```diff
@@ -0,0 +1,164 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"log"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"github.com/golang/dep"
+	"github.com/golang/dep/internal/gps"
+	"github.com/golang/dep/internal/test"
+	"github.com/pkg/errors"
+)
+
+func TestVndrConfig_Import(t *testing.T) {
```

Comment:
No problem, this was pretty easy to do.

There's now a lot of repeated code between the three importers, though. I've got a sketch of how this could be resolved in https://github.com/spenczar/dep/compare/vndr_importer...importer_test_refactor. It makes things a more abstract, which is bad, but it reduces the code duplication, which is good. If you think this is a net positive, let me know and I'll make an actual PR - up to you, I can see it either way.